libs/governance/src/lib/proposals-hooks/Proposal.graphql

```diff
@@ -43,3 +43,12 @@ subscription OnUpdateNetworkParameters {
     }
   }
 }
+
+query ProposalOfMarket($marketId: ID!) {
+  proposal(id: $marketId) {
```

Comment:
suggestion (non-blocking): I'm fairly confident that the ids match but if you want to be extra safe here then you can get the proposal of the market from market itself:
```gql
market(id: $marketId) {
  proposal {
    id
    terms {
      enactmentDatetime
    }
  }
}
```

Reply:
I didn't want to add extra fields to the big already market query, especially when we need proposal data only after user action like hover on element for display a tooltip